README.md

```diff
@@ -53,18 +53,23 @@ user1 = client.feed('user', '1', 'FEED_TOKEN');
 // Get activities from 5 to 10 (slow pagination)
 user1.get({limit:5, offset:5}, callback);
 // (Recommended & faster) Filter on an id less than a given UUID
-user1.get({limit:5, id_lt:"e561de8f-00f1-11e4-b400-0cc47a024be0"}, function(error, response, body) { /* callback */ });
+user1.get({limit:5, id_lt:"e561de8f-00f1-11e4-b400-0cc47a024be0"});
+
+// All API calls are performed asynchronous and return a Promise object
+user1.get({limit:5, id_lt:"e561de8f-00f1-11e4-b400-0cc47a024be0"})
+	.then(function(body) { /* on success */ })
+	.catch(function(error) { /* on failure */ });
 
 // Create a new activity
 activity = {'actor': 1, 'verb': 'tweet', 'object': 1, 'foreign_id': 'tweet:1'};
-user1.addActivity(activity, function(error, response, body) { ... });
+user1.addActivity(activity);
```

Comment:
APIs return ID of the inserted activity. it would be great to add an example that shows that